compact/tracker.xml

```diff
@@ -28,7 +28,7 @@
     <comment> barrel length: 2R/tan(angle) - dz </comment>
 
     <comment> Position in center of space behind DIRC </comment>
-    <constant name="OuterMPGDBarrel_rmin"            value="EcalBarrel_rmin - BarrelExtraSpace_thickness / 2."/>
+    <constant name="OuterMPGDBarrel_rmin"            value="CentralTrackingRegion_rmax"/>
```

Comment:
Do I understand correctly that this is a temporary fix to put the outer MPGD layer in the position of DIRC, so when the DIRC is properly implemented and put back the `OuterMPGDBarrel_rmin` will be shifted outwards after the DIRC?

Reply:
Yeah, my initial question exactly, but I hadn't had a chance to really look into it. As the designated dirc implementer, I'd like to know.

Reply:
@mariakzurek You are absolutely right, this is temporary.

The main focus of this PR is to allow the current outer barrel MPGD geometry to exist at any radii (the current design fails the extrusion check, it could be that the current radius was fudged to avoid this). The fix needed is to set `rc` correctly, as it is already done for the inner MPGD and the inner tracker pieces. I also move the outer MPGD out of barrel region, so that it does not block the work on barrel calorimetry.

Reply:
This was done under assumption that DIRC is gone for this campaign. And I'm, of course, open to suggestions regarding nominal radius for MPGD.

Reply:
I think for this simulation campaign, we should look to have at least a passive cylindrical layer that approximates the average DIRC material since this will affect the tracking and calorimeter performances.

Reply:
We will have a working DIRC for this campaign: i.e. a DIRC that is dimensionally correct and has the correct material in the shape of DIRC bars.

Reply:
I think that for the purpose of this PR (removing MPGD from the Calorimeter, and fixing the calorimeter r position and parametrization of DIRC and MPGD positions), we can either leave the `OuterMPGDBarrel_rmin` at `CentralTrackingRegion_rmax` temporarily, as in the current geometry the DIRC is not implemented, and then move it after DIRC as @mposik1983 suggested. Or fix it to `CentralTrackingRegion_rmax+BarrelPIDRegion_thickness`, but the `BarrelPIDRegion_thickness` would need to be changed (from 8 cm to 3 cm), so that `OuterMPGDBarrel_rmin = 74.5 cm`.